sklearn/linear_model/tests/test_ridge.py

```diff
@@ -487,6 +487,35 @@ def test_class_weights():
     assert_array_almost_equal(clf.intercept_, clfa.intercept_)
 
 
+def test_class_weight_vs_sample_weight():
```

Comment:
Something similar in #4215 coming soon enough too, I imagine, for the rest of the ensemble tests.